index.html

```diff
@@ -0,0 +1,259 @@
+                    <p class="art_print_paragraph">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. Has been the industry's standard.
+                    </p>
+                    <ul class="art_print_btn_container">
+                        <li><button class="transparent-btn">HTML</button></li>
+                        <li><button class="transparent-btn">Bootstrap</button></li>
+                        <li><button class="transparent-btn">Ruby</button></li>
+                    </ul>
+                </div>
+                <div class="art-main-btn-container">
+                    <button class="primarybtn">See Project</button>
+                </div>
+            </div>
+
+            <div class="professional_art_printing-section-items">
+                <div class="art_print_container">
+                    <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                    <p class="art_print_paragraph">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. Has been the industry's standard.
+                    </p>
+                    <ul class="art_print_btn_container">
+                        <li><button class="transparent-btn">HTML</button></li>
+                        <li><button class="transparent-btn">Bootstrap</button></li>
+                        <li><button class="transparent-btn">Ruby</button></li>
+                    </ul>
+                </div>
+                <div class="art-main-btn-container">
+                    <button class="primarybtn">See Project</button>
+                </div>
+            </div>
+
+            <div class="professional_art_printing-section-items">
+                <div class="art_print_container">
+                    <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                    <p class="art_print_paragraph">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. Has been the industry's standard.
+                    </p>
+                    <ul class="art_print_btn_container">
+                        <li><button class="transparent-btn">HTML</button></li>
+                        <li><button class="transparent-btn">Bootstrap</button></li>
+                        <li><button class="transparent-btn">Ruby</button></li>
+                    </ul>
+                </div>
+                <div class="art-main-btn-container">
+                    <button class="primarybtn">See Project</button>
+                </div>
+            </div>
+        </section>
+    </div>
+
+    <!-- DARK SECTION -->
+    <section class="dark_section">
+        <!-- ABOUT-ME SECTION -->
+        <section class="aboutme_section">
+            <div class="aboutme_container">
+                <h1 class="aboutme_header_text">About me</h1>
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/Peter-Dumbari/Portofolio-Microverse/pull/2#discussion_r1148405178), please make sure to add the `font-weight` and `font-size` according to the design given in Figma template.✨ As an example, you can use something like below
```
    font-weight: 500;
    font-size: 4.3rem;
```
- [ ] As suggested, please consider using `h2` instead of `h1` for the about section title

### _Your design before 🔽_
![image](https://user-images.githubusercontent.com/106119105/227744914-e84a96cc-8093-44d1-b8e2-6e40729fe788.png)


### _Your design now 🔽_
![image](https://user-images.githubusercontent.com/106119105/227744955-420dd666-236c-4742-8b1d-7081b1079e3c.png)

### _Figma design 🔽_
![image](https://user-images.githubusercontent.com/106119105/227744945-a883c789-7d91-45ad-919b-af9b0b3a6e22.png)